safe/impact_functions/core.py

```diff
@@ -674,8 +658,9 @@ def get_admissible_plugins(keywords=None):  # , name=None):
 
 
 def parse_single_requirement(requirement):
-    '''Parse single requirement from impact function's doc to category,
-        subcategory, layertype, datatype, unit, and disabled.'''
+    """Parse single requirement from impact function's doc to category,
```

Comment:
Yes, of course.

I just change the single quote to double quote. We will remove this function soon we remove the docstring. It's 80% finished. so, I will just leave it now for now